PlutoHelperAgent/SharedFunctions.m

```diff
@@ -0,0 +1,252 @@
+//
+//  SharedFunctions.m
+//  PlutoHelperAgent
+//
+//  Created by Dave Allison on 05/03/2018.
+//  Copyright © 2018 Guardian News & Media. All rights reserved.
+//
+
+#import "SharedFunctions.h"
```

Comment:
@Dala0 oh, could you also rename this module to something more descriptive like "ProjectLockerCommunicator" or "ProjectLockerLogin"? "SharedFunctions" is kinda meaningless

Reply:
Done